pkg/kapp/cmd/tools/diff_flags.go

```diff
@@ -35,4 +36,6 @@ func (s *DiffFlags) SetWithPrefix(prefix string, cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&s.Mask, prefix+"mask", true, "Apply masking rules")
 
 	cmd.Flags().BoolVar(&s.AgainstLastApplied, prefix+"against-last-applied", true, "Show changes against last applied copy when possible")
+
+	cmd.Flags().StringVar(&s.Filter, prefix+"filter", "", "Set changes filter (example: {\"and\":[{\"ops\":[\"update\"]},{\"existingResource\":{\"kinds\":[\"Deployment\"]}]})")
```

Comment:
nit, instead of `"` you can use `, so that you dont have to escape "

Reply:
Done